app/helpers/search_helper.rb

```diff
@@ -20,7 +20,7 @@ def render_logo(id, kind)
     return if key == 'ignored'
 
     info = t("search.accreditations.items.#{key}")
-    link_to "glossary/##{key}", class: "accreditation t-#{kind}" do
+    link_to "/#{locale}/glossary##{key}", class: "accreditation t-#{kind}" do
```

Comment:
I would use named routes and not use raw links like that. If you want we can pair on it.